lib/doorkeeper/config.rb

```diff
@@ -269,6 +269,9 @@ def configure_secrets_for(type, using:, fallback:)
     option :grant_flows,                    default: %w[authorization_code client_credentials]
     option :handle_auth_errors,             default: :render
 
+    # Hook to allow arbitrary user-application access enforcement
+    option :application_access,             default: ->(_application, _resource_owner) { true }
```

Comment:
I would like to rename the option. Maybe something like `allow_resource_owner_to_access_application`, but shorter :grimacing: Can't same exactly, need to think about it

Reply:
I changed it to `resource_owner_application_access`, let me know if you still want to change it